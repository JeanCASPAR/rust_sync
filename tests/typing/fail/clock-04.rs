use rust_sync::sync;

sync! {
    node oui() = ()
    where
        a : float = 3.0 whennot a;
}

fn main() {}
