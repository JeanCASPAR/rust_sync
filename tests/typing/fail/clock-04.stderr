error: proc macro panicked
 --> tests/typing/fail/clock-04.rs:3:1
  |
3 | / sync! {
4 | |     #![pass(1)]
5 | |
6 | |     node oui() = ()
7 | |     where
8 | |         a : float = 3.0 whennot a;
9 | | }
  | |_^
  |
  = help: message: not yet implemented: raise error: clock not boolean
